library/lua/dfhack.lua

```diff
@@ -166,13 +166,9 @@ end
 
 DEFAULT_NIL = DEFAULT_NIL or {} -- Unique token
 
-function defclass(...)
-    return require('class').defclass(...)
-end
-
-function mkinstance(...)
-    return require('class').mkinstance(...)
-end
+local class = require('class')
```

Comment:
copying my discord message here for posterity:

The reason for these indirections in dfhack.lua is to avoid circular dependencies.  dfhack.lua must not load any other files when it is parsed for the first time.

dfhack.lua is loaded first at Lua context initialization time. If it calls `require` to load another module, that module may try to use symbols in dfhack.lua that are not loaded yet.